src/team/g1/DemoG1.java

```diff
@@ -11,12 +11,12 @@ public static void main(String[] args) {
 
 		System.out.println("Started from the bottom now we here");
 
-		System.out.println("RIP to ma' Popeyes!");
-		
 		System.out.println(" Hanan");
 
 		// User Hanan
 		System.out.println(" we are having fun with github ");
+		System.out.println("I was hungry so I ate a snack");
```

Comment:
I wanted a snack too. anyway, i guess I'll approve it even though you didn't share.